src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -77,6 +78,12 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->defaultValue(['%kernel.project_dir%/assets/styles/app.scss'])
                 ->end()
+                ->fixXmlConfig('load_path')
```

Comment:
This is broken. `fixXmlConfig` is a method of the node, not of the NodeBuilder returned by `children()`

Reply:
Oh sorry I missed that. Also, should `root_sass` be fixed too?

Reply:
`root_sass` will be hard to handle as it is not a plural name for which we might make the repeated XML tag use the singular as name...

Reply:
Alright. I just fixed my fix.